server/api/impl/proxy/asset.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package proxy
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/pkg/errors"
+)
+
+func (p *Proxy) Asset(appID api.AppID, assetName string) ([]byte, error) {
+	app, err := p.store.LoadApp(appID)
+	if err != nil {
+		return nil, errors.Wrapf(err, "can't load app - %s", appID)
+	}
+	for _, asset := range app.Manifest.Assets {
+		if asset.Name == assetName {
```

Comment:
nit: flattening the "arrow code" (https://blog.codinghorror.com/flattening-arrow-code/) and maybe replacing the ifs with a switch